docs/_documentations/image-registry-credentials.md

```diff
@@ -18,36 +18,31 @@ Before you proceed, see the [examples](#examples) section on recommended values
 
 ## Examples:
 The following deployment registries have been tested and verified with Codewind:
-- Dockerhub:
-    - Host: `docker.io`
+- Docker Hub:
+    - Address: `docker.io`
     - Codewind deployment registry: `docker.io/<username>`
```

Comment:
This needs to be `Namespace` instead of `Codewind deployment registry`. Value can be `<namespace>`

Reply:
You may use this screenshot

![Screen Shot 2019-12-16 at 12 47 18 PM](https://user-images.githubusercontent.com/31771087/70930204-9c264a80-2002-11ea-86e5-1e5f5e0a64eb.png)

Reply:
I'm wondering if we want to clarify that `<namespace>` will likely be your username on Docker Hub?

Reply:
or we can use this above screenshot with doc https://github.com/eclipse/codewind/issues/1488#issuecomment-566170812 since https://www.eclipse.org/codewind/openshiftregistry.html needs to be updated

Reply:
@maysunfaisal and @johnmcollier, I'll make these changes and save the screenshot for 1488.

Reply:
@j-c-berger thx. Maybe in the namespace field, can we add a hint in parenthesis like John suggested?

for Docker Hub the `Namespace` value is most likely to be the username.

Reply:
@maysunfaisal, yep, can do!